Remotion/Web/Development.WebTesting/Utilities/RetryUntilTimeout.cs

```diff
@@ -55,27 +56,58 @@ public void Run ()
       _retryUntilTimeout.Run();
     }
 
-    public static void Run ([NotNull] Action action)
+    /// <summary>
+    /// Executes a given <see cref="Action"/> repeatedly (using the given retry interval) until no exception is thrown during execution or until the
+    /// given timeout has been reached (in which case the final exception is rethrown).

```

Comment:
last instead of final. Please check rest of new docs.

Reply:
I adopted the wording from the existing class documentation, which uses "final". Using "last" would introduce inconsistencies. Do you want me to change all such instances to "last" or keep it as it is?

Reply:
I'm going with option c), please fix the xmldocs on the type to consistently use "last".